src/App.js

```diff
@@ -0,0 +1,84 @@
+import './App.css';
+import { nanoid } from 'nanoid';
+import { useState, useEffect } from 'react';
+import List from './components/List';
+
+const getLocalStorage = () => {
+  let list = localStorage.getItem("list")
+  if (list) {
+    return (list = JSON.parse(localStorage.getItem("list")))
+  } else {
+    return []
+  }
+}
+function App() {
+  const [name, setName] = useState("")
+  const [list, setList] = useState(getLocalStorage())
+  const [isEditing, setIsEditing] = useState(false)
+  const [editID, setEditiID] = useState(null)
```

Comment:
There is an extra `i` in the updater function for the editID.
```suggestion
  const [editID, setEditID] = useState(null)
```